tests/sentry/sentry_metrics/test_gen_metrics_multiprocess_steps.py

```diff
@@ -408,12 +408,7 @@ def test_process_messages_default_card_rollout(set_sentry_option: Callable[...,
 
     outer_message = Message(Value(message_batch, last.committable))
 
-    with set_sentry_option(
-        "sentry-metrics.cardinality-limiter.orgs-rollout-rate",
-        1.0,
```

Comment:
1.0 was the default, but it was also not used.

Reply:
IIRC there was a test that used this option? Should we modify that test accordingly?

Reply:
This is the only test and it set this option to the default. I suppose this change is all we need?